sklearn/utils/_indexing.py

```diff
@@ -507,7 +519,6 @@ def resample(*arrays, replace=True, n_samples=None, random_state=None, stratify=
 
     if len(arrays) == 0:
         return None
-
```

Comment:
Please do not introduce unrelated formatting changes to focus the diff of the PR to the scope of the PR.